agent/app/agent_capability.go

```diff
@@ -111,6 +133,10 @@ const (
 func (agent *ecsAgent) capabilities() ([]*ecs.Attribute, error) {
 	var capabilities []*ecs.Attribute
 
+	for _, cap := range nameOnlyAttributes {
+		capabilities = appendNameOnlyAttribute(capabilities, cap)
```

Comment:
you could also reduce a bit more if you wanted:

```
capabilities = appendNameOnlyAttribtue(capabilities, attributePrefix + cap)

Reply:
Yes, that would be neat